server_apps/data_transfer/zfishbook/zfishbook.pl

```diff
@@ -154,14 +154,15 @@ ($)
 
 print "\n\nStarting to load ...\n\n\n" if $doTheLoad > 0;
 
-if ($doTheLoad > 0)[
```

Comment:
I am not sure what the difference is.
Just indentation?

Reply:
For some reason, the old code was using a square bracket '[' instead of curly braces '{'.  It's a syntax error so I'm not sure how it could have ever worked.